proposal-membership.md

```diff
@@ -36,3 +36,11 @@ The other disjoint groups shall create new consortiums with different names.
 Recognition is based on mutual consent:
   1. Active members are encouraged, but not obliged, to list “Past Members” and a summary of their contributions on the consortium’s public website for posterity.
   2. “Past Members” have the possibility, but no obligation, to be listed publicly on the consortium website. Their active consent will be sought by active members that want to recognize them, unless the past members are not able to give that consent anymore. In the latter case, the choice of listing or not the “Past Member” is left to the best judgement of the active members.
+
+  ## Modifying the Membership (this document)
+  
+  The '(Y)es/(W)hatever/(?)Clarification/(!)Alternative Suggestion' decision process, explained in the [Voting Process](./voting-process.md) document, is used to make modifications or complete rewrites of this document. 
+  
+  Modifications that provide clarifications without changing the meaning of sections of this document are performed according to the [Voting Process](./voting-process.md). Modifications that do change the meaning of this document require the following more stringent criteria to be accepted, in addition to the usual requirements for regular proposals:
+  1. At least 50% + 1 of active members must have voted 'Yes' to the proposal.
```

Comment:
I wonder if these extra criteria maybe just add unnecessary complexity? I.e. I don't think they change the process enough to warrant having them.

Reply:
@Powersource I personally see Membership as one of the core pillars of the "Constitution" of the organization. The current Voting Process only requires a single Yes for a change to pass. Moreover, everyone's vote default to 'Whatever' at the deadline, so we could really end with a single person making Membership changes.

I do agree however, that having different criteria in the Membership document adds complexity. So I am open to other arguments that would substantiate how this is unnecessary.

Reply:
In a healthy-ish community, if it's a major change then it would get talked about a lot and most people in the org would look at and consider the change. In that case it's not 1 yes, it's consent from pretty much everyone in the org. And just like it technically only requires 1 yes, it only requires 1 ! to block.

Reply:
@Powersource I can get behind your argument and we would always have the option of changing it later if we find a problem.